console/DOCKER_HUB.md

```diff
@@ -0,0 +1,81 @@
+# Quick reference
+
+-    **Maintained by**:  
+      [Georchestra.org](https://www.georchestra.org/)
+
+-    **Where to get help**:  
+     the [Georchestra Github repo](https://github.com/georchestra/georchestra), [Matrix chat](https://matrix.to/#/#georchestra:libera.chat), Stack Overflow
+
+# Featured tags
+
+- `latest`, `23.0.x`, `23.0.5`
+
+# Quick reference
+
+-	**Where to file issues**:  
+     [https://github.com/georchestra/georchestra/issues](https://github.com/georchestra/georchestra/issues)
+
+-	**Supported architectures**:   
+     [`amd64`](https://hub.docker.com/r/amd64/docker/)
+
+-	**Source of this description**:  
+     [docs repo's `console/` directory](https://github.com/georchestra/georchestra/blob/master/console/DOCKER_HUB.md)
+
+# What is `georchestra/console`
+
+**Console** is a module for geOrchestra which allow:
+- public users to create accounts/reset their passwords.
+- administrators to manage users, organisations and roles.
+- registered users to edit their account details.
+
+A [complete description](https://github.com/georchestra/georchestra/blob/master/console/README.md) is available on github .
+
+# How to use this image
+
+As for every other geOrchestra webapp, its configuration resides in the data directory ([datadir](https://github.com/georchestra/datadir)), typically something like /etc/georchestra, where it expects to find a console sub-directory.
+
+You can run the image using :
+```shell
+docker run -v georchestra_datadir:/etc/georchestra georchestra/console:latest
+```
+
+Or with `docker compose`: 
```

Comment:
It's `docker-compose.yml` not `docker compose`.
`docker compose` is the command and `docker-compose.yml` is the filename for a docker compose project.

Reply:
I did understood what you wanted to say but here you present a structure of a file not a command.

Hence it makes more sense to call it `docker-compose.yml`.